commons/commons-lang/src/main/java/com/clicktravel/common/validation/Check.java

```diff
@@ -168,4 +169,23 @@ public static void matchesPattern(final String field, final String value, final
             throw new ValidationException(errorMessage, field);
         }
     }
+
+    /**
+     * Check that a string contains a valid value from the enumerated class and return that enumerated value
+     * @param field Name of field to report if check fails
+     * @param value Containing string to check
+     * @param enumClass The enumeration class containing all valid values
+     * @throws ValidationException if check fails
+     * @return enumerated conversion of value
+     */
+    public static <E extends Enum<E>> E validateToEnum(final String field, final String value,
```

Comment:
This method feels a bit weird being in the `Check` class as it is actually doing the conversion if the value is valid rather than just doing the check.  I think it either needs to simply perform the check as to whether the value is valid for the given enum or be moved to a different class (sadly I can't see any existing candidates which jump out at me currently).

I think something like `isValidEnumValue` would be a better name in terms of describing the check that is being performed.